src/artifact_choosing.rs

```diff
@@ -1,5 +1,12 @@
-#[cfg(target_os = "windows")]
-static PLATFORM_KEYWORDS: &[&str] = &["win32", "win64", "windows"];
+#[cfg(all(target_os = "windows", target_arch = "x86"))]
```

Comment:
I think we could use `target_pointer_width` instead. In practice it probably does not change much but the intent is a little clearer 🙂 

```suggestion
#[cfg(all(target_os = "windows", target_pointer_width = "32"))]
```